cookbooks/aws-parallelcluster-platform/test/controls/chrony_spec.rb

```diff
@@ -9,7 +9,7 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-control 'chrony_installed_and_configured' do
+control 'tag:install_chrony' do
   title 'Test chrony installation and configuration'
 
   only_if { !os_properties.redhat_ubi? }
```

Comment:
question, non-blocking: Is there a way to test the installation of `npt*` packages?

Reply:
We can take a look later.